docs/guides/deploy_service.md

```diff
@@ -198,8 +206,9 @@ We illustrate the full local deployment workflow using the `hello_world` service
             apiVersion: storage.k8s.io/v1
             kind: StorageClass
             metadata:
-              name: nfs-ephemeral
-            provisioner: k8s.io/minikube-hostpath 
+                name: nfs-ephemeral
+            provisioner: kubernetes.io/no-provisioner
+            volumeBindingMode: WaitForFirstConsumer
```

Comment:
Is this reflected on the autonomy command itself? I.e. will generated deployments include this.

Reply:
Tom suggested not to update the default templates https://github.com/valory-xyz/open-autonomy/issues/1849